### DISCUSSION TITLE
Second Brain

### DISCUSSION THREAD

**Discussion by seekanddefine** [athensresearch/athens#1072] [General]
Did you know the human body has two brains? 

I noticed that on Obsidian's Twitter, they have advertised:
A second brain, for you, forever. Join our community.

![image](https://user-images.githubusercontent.com/38668794/116335804-0e817000-a79d-11eb-8653-0dcb7cf75965.png)

A thought I'd start a discussion about using the term or moving away from it, since I've heard it fairly often since joining. 

It's interesting to note that the enteric nervous sytem  also known as the "second brain" of humans, operates independently of the brain, spinal cord, and central nervous system. You have millions of neurons throughout your entire body including places like the gastrointestinal tract. All of these neurons coordinate their signals, so your muscles contract correctly to digest food and process waste.  How many evolutions of nervous systems occurred before we could do this as a species? 

I love the idea of the second brain and would love to expand on it. What if there was a view on my knowledge graph of everything I'd ever taken notes on that showed up organized into the image of an actual brain? A display of a 3D neural brain-shaped network of maps. I'd download that app whether I was going to use it or not just for fun.  Superimposed somehow? How do the branches we seek to grow and connect as effectual notetakers mirror what is happening in the network of our neurons? 

Building a Second Brain: The Illustrated Notes after taking a course by Tiago Forte. He uses Roam and Keyboard Maestro. https://maggieappleton.com/basb[](https://maggieappleton.com/basb) 

There is a very large group of digital note-taking nerds on Twitter. What do the groups look like on the rest of social media? Facebook groups? IG? etc.

It doesn't look copyrighted. Not sure on phrase/slogan copyright stuff. 

Video on Building a Second Brain Course completer. https://www.youtube.com/watch?v=OP3dA2GcAh8[](https://www.youtube.com/watch?v=OP3dA2GcAh8) 

Using the term knowledge worker.

**Comment by Limezy** [comment]
Hi @seekanddefine, 

Interesting discussion. On my side, I'm not 100% sure I like this "second brain" thing. Let me try to give a few reasons here : 

### Strong link to Tiago Forte
Although it doesn't look like a copyrighted concept, that semantic is highly linked with Tiago Forte. Do we really want that ?
Athens is not a tool that was specifically designed to support Tiago's concepts and methods.

### Means nothing for beginners, doesn't add anything for experts
"Second brain" is a well-known concept among note-taking or productivity geeks, just like Zettelkasten, Evergreen Notes, "PKM"... but do we really need to use these terms ? Obviously the guys that are confortable with these concepts may be the first target of Athens, as we could see them as easy converts (already convinced users of Roam or equivalent tool having backlinks, not convinced about Roam subscription model or pricing...). But these don't need bullshit headlines like "we will help you build a second brain", which doesn't tell anything about Athens. They need to understand in one or two punchlines how Athens is different and better than what they already know or use. Something like "open source, local-first, self-hostable note-taking outliner app featuring backlinks" would be much more convincing for them.

However, obviously Athens should have a broader view than just targeting a few note-taking nerds (that change note-taking tool very year anyway). So let's have a look on what could "Second brain" mean for a complete note-taking app beginner ? Absolutely **nothing**. I've tried to deep-dive into the skin of such a guy. He finds out Athens, searches "Second brain app" on Google, and what does he find ? 
* Complicated Reddit posts with crazy words like Zettelkasten, Evergreen Notes, "PKM"
* 30 minutes videos about guys arguing on what is the best folder structure to keep your notes in Obsidian
* Link to Tiago's website featuring 4 figures course on "how to build a second brain" 

You get my point : we don't drive the potential future user where we want. We've just lost him in the beginning of a long journey to try understand what the heck all this is about. Even worse, since Athens is so new in the field, none of the tutorials / videos / articles he is about to read are about Athens. If we are lucky, he'll come back. But he may not.

### It's marketing bullsh*t anyway
Who really thinks a software - even a beautifully crafted one - would be in any way something close to the human brain ?
I would prefer if Athens advertised on what it is really, and that is not a brain (first, second, third or whatever the number). 
It's just a note taking app with powerful **_organic concepts_** under the hood (links, backlinks, graph view...). But more on that LATER, since this is the "what", and it's always a mistake to "Start with what".

### "Start with why", and then only explain the what
I'm pretty sure you know this, if not you can google it and I recommend you have a look at the Ted conference (no need to buy the book, it's the same stuff over and over). The take away of that conference is very simple yet powerful : for successful marketing, just start with why. As explained above, we mainly have two target audiences there (I'm not sure about which order should be used)
For all examples below, _**I've put the why in bold italics**_, and the what in normal fonts

**The productivity nerds / PKM experts.** 
For them, the "Why" is pretty simple : we need to explain **why** Athens is better than all the other tools they already know and / or use. IMHO our "Why" should include all of these : 
* **_You are tired of heavy subscription models or anxious about the resilience of your current tool ?_** Athens is open source
* **_You need privacy and safety for your data ?_** Athens is self-hosted
* **_You need real-time collaboration ?_** Athens has a built-in function for multiple users graphs
* **_You need native apps and speed of use ?_** Athens features a local-first approach

**The complete beginners.**
For them, the "Why" is more general and could maybe drive them to "competitors", but that's not the point.
The bigger the modern note-taking app users pie is, the better for everybody. Some will quit for Logseq, others will quit for Obsidian, but that's not important as others will just do the other way around.
* **_You keep loosing your notes ?_** Athens will help you naturally organize them in one place using the link concept
* **_You never build anything from them ?_** Athens will add value to them by organically connecting your ideas

My example above are very rough, and I don't know how they should be marketed, but the main point is : we should explain to our different targets why an Athens-like software is something they didn't know they needed (beginners), and why Athens is the best tool in town for their PKM needs (experts).

### Conclusion

So, after having exposed the "why" to complete beginners, I'm not against using the brain analogy to explain why a linked notes app like Athens feels so easy and natural to use. But just dropping the concept of a "second brain" as a headline doesn't feel right to me.

This is obviously only my personal opinion !

**Reply by jefftangx** [reply]
I agree with your entire post more or less. I will add that we are currently defining our value proposition, target user, and target use case as part of #1099. I'm planning to synthesize Athens's unique differentiation this week, which may be quite different from the other networked tools for thought. The teaser right now is that we will focus on a few underserved areas in the entire space: research, collaboration, and accessibility. By doing something quite different from the others (Roam, Obsidian, LogSeq, etc.), the differentiation will be so significant that we wouldn't compare or choose between Athens and the others.

Re: "Second Brain", I'm okay with it for now. I think it may be true that Tiago created or popularized it, but I don't think he tries to own it. It helps that he preaches tool-agnosticism ([and doesn't really pick favorites](https://twitter.com/fortelabs/status/1377505949221445633/photo/1)). Personally, I use "Second Brain" or "knowledge graph", even though they are truly descriptive, because what we are is still emerging. One part that makes this more difficult is that most of the current tools are trying to do everything for everyone as a general-purpose technology. I think it can be clearer if we focus on specific jobs such as research or collaboration or target specific markets like for researchers, developers, or mainstream consumers. More to come here soon! @Limezy

**Comment by zhangxiubo** [comment]
I feel the term is fine as a marketing slogan; it sounds smart.  Upon hearing the term "second brain", people who are not familiar with the concept of PKM will probably wonder what it means, so it certainly leverages the curiosity factor.
